src/components/Viewport.ts

```diff
@@ -1443,10 +1434,27 @@ export default class Viewport {
         next: 0,
       };
     } else if (this.canSetBoundMode()) {
-      state.scrollArea = {
-        prev: firstPanel.getPosition(),
-        next: lastPanel.getPosition() + lastPanel.getSize() - state.size,
-      };
+      const sumOriginalPanelSize = lastPanel.getPosition() + lastPanel.getSize() - firstPanel.getPosition() + options.gap;
```

Comment:
@WoodNeck 

Why add options.gap?

Reply:
That's my mistake, thanks for checking it.